maliput/tests/junction_tests.rs

```diff
@@ -34,11 +34,11 @@ fn junction_api() {
     let road_network = common::create_t_shape_road_network();
     let road_geometry = road_network.road_geometry();
     let junction_id = String::from("0_0");
-    let junction = road_geometry.get_junction(&junction_id);
+    let junction = road_geometry.get_junction(&junction_id).unwrap();
     assert_eq!(junction.road_geometry().id(), road_geometry.id());
     assert_eq!(junction.id(), junction_id);
     let num_segments = junction.num_segments();
     assert_eq!(num_segments, 1);
-    let segment = junction.segment(0);
+    let segment = junction.segment(0).unwrap();
```

Comment:
Can we add a test for when segment index is wrong?

Reply:
Done.